modules/ROOT/pages/db/database-connector-examples.adoc

```diff
@@ -336,8 +336,7 @@ the following operation inserts a new record in the `electronic` table:
 </db:insert>
 ----
 
```

Comment:
INSTRUCTIONS FOR EDIT

Hi Kim, Can you please review this PR (green only--it was previously edited by you). Not urgent. By end of Monday would be fine.